src/base_model.jl

```diff
@@ -395,10 +451,12 @@ function calculate_demand(ECModel::AbstractEC)
     time_res = profile(ECModel.gen_data, "time_res")
     energy_weight = profile(ECModel.gen_data,"energy_weight")
 
-    data_load = Float64[sum(sum(
-                profile_component(users_data[u], l, "load") .* time_res .* energy_weight)
-                for l in asset_names(users_data[u], LOAD)
-            ) for u in user_set]
+    data_load = Float64[sum(
+        sum((profile_component(users_data[u], l, "load") + ECModel.results[:P_adj_tot_us][u, :])
+        .* time_res .* energy_weight)
+        for l in asset_names(users_data[u], LOAD))
+        for u in user_set
+        ]

```

Comment:
```suggestion
    data_load = Float64[sum(
        sum(ECModel.results[:P_L_tot_us][u, :]) .* time_res .* energy_weight) for u in user_set
        ]

```

Probably we can revise to completely rely on this expression; this applies also in other instances.
Great idea to have a total by user!